Tennis-scheduler/src/main/java/com/tennis/tennisscheduler/controllers/AuthenticationController.java

```diff
@@ -5,9 +5,13 @@
 import com.tennis.tennisscheduler.dtos.UserTokenStateDto;
 import com.tennis.tennisscheduler.mappers.PersonDtoMapper;
 import com.tennis.tennisscheduler.models.Person;
+import com.tennis.tennisscheduler.models.Timeslot;
+import com.tennis.tennisscheduler.services.TimeslotService;
```

Comment:
If you are not using it, delete it

Reply:
I agree, I don't see any crucial changes in this class, maybe you added something and then changed your mind. :)
Just clean up the leftover 👍